tests/functional/ext/docparams/missing_param_doc.py

```diff
@@ -207,3 +211,31 @@ def foobar19(one, two, **kwargs):
     """
     print(one, two, kwargs)
     return 1
+
+
+class Word:
+    """
+    Methods decorated with `typing.overload` are excluded
+    from the docstring checks. For example: `missing-param-doc` and
```

Comment:
Technically there are other docstring checks.

```suggestion
    from the docparam checks. For example: `missing-param-doc` and
```